### DISCUSSION TITLE
Make cpEditUrl available in GraphQL

### DISCUSSION THREAD

**Discussion by Wiejeben** [craftcms/cms#9573] [Ideas]
As a service we would want to add an edit button to on our pages, for this we would like to have the ability to get the value from the `cpEditUrl` function via GraphQL.

**Comment by brandonkelly** [comment]
This would obviously reveal the CP URL, which may not be desired, so it’s something we’d need to add a new schema setting for, similar to what we do for drafts and revisions, and have it be disabled by default.

**Reply by Wiejeben** [reply]
Ah I didn’t think about that! Will probably look into extending GraphQL ourselves since our GraphQL queries are handled internally.

**Reply by andris-sevcenko** [reply]
It shouldn't be too hard if you use https://craftcms.com/docs/3.x/extend/graphql.html#modifying-type-fields

Also, sections are memoized, so there probably won't be a noticeable performance hit, either.
